lib/auth/widget/suggested_space_card_widget.dart

```diff
@@ -0,0 +1,265 @@
+import 'dart:ui';
+
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:totem_app/api/models/event_detail_schema.dart';
+
+/// Card used in Suggestions tab, built from EventDetailSchema.
+class SuggestedSpaceCard extends StatelessWidget {
+  const SuggestedSpaceCard({
+    required this.event,
+    super.key,
+  });
+
+  final EventDetailSchema event;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final backgroundImage = event.space.image;
+    final timeLabel = _buildTimeLabel(event.start);
+    final title = event.title.isNotEmpty ? event.title : event.spaceTitle;
+    final keeperName = event.space.author.name ?? 'Keeper';
+    final seatsLeft = event.seatsLeft;
+
+    return Container(
+      height: 120,
+      width: double.infinity,
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(20),
+        image: backgroundImage == null
+            ? null
+            : DecorationImage(
+                image: NetworkImage(backgroundImage),
+                fit: BoxFit.cover,
+              ),
+        color: backgroundImage == null
+            ? theme.colorScheme.primaryContainer
+            : null,
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Expanded(
+            flex: 2,
+            child: Padding(
+              padding: const EdgeInsets.all(10),
+              child: ClipRRect(
+                borderRadius: BorderRadius.circular(20),
+                child: BackdropFilter(
+                  filter: ImageFilter.blur(sigmaX: 2, sigmaY: 2),
+                  child: Align(
+                    alignment: Alignment.topLeft,
+                    child: IntrinsicWidth(
+                      child: Container(
+                        padding: const EdgeInsets.symmetric(
+                          horizontal: 10,
+                          vertical: 5,
+                        ),
+                        decoration: BoxDecoration(
+                          color: const Color.fromRGBO(38, 47, 55, 0.60),
```

Comment:
Do we have a place to define colors? I've seen these one-off colors cause a lot of headaches over time.